.gitignore

```diff
@@ -21,3 +21,4 @@ docs/build/
 .coverage
 cover
 env
+.idea
```

Comment:
This isn't produced by anything in our repository. It should be ignored globally on your system.

See: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore